src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+  background-color: #fff;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.menu {
+  display: flex;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.menu-li {
+  display: flex;
+}
+
+.link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  padding: 23px 0;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
```

Comment:
they are the same, it's better to move them into common selector. also it's a good practice to move colors into variables